platform-api/src/routers/platform.py

```
@@ -0,0 +1,105 @@
+        
+    async with get_client() as client:
+        resp = await client.post(hook_url)
+        success = 200 <= resp.status_code < 300
+        return RebuildResponse(
+            site=site, 
+            triggered=success, 
+            message="Rebuild triggered" if success else f"Rebuild failed: {resp.status_code}"
+        )
+
+@router.get("/health")
+async def aggregated_health(settings: WorkerSettings = Depends(get_settings)):
```

Ruff (0.15.12)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)